### ISSUE TITLE
[TEST] Use `app.inject.query` parameter instead of `qs.stringify`

### ISSUE THREAD

**Issue by spaenleh** [graasp/graasp#1127]
> By the way, there is a query property for the `app.inject` in our test file context. So we shouldn't use these `qs.stringify` function

_Originally posted by @Defferrard in https://github.com/graasp/graasp/pull/1126#pullrequestreview-2142026815_

**Comment by Defferrard** [comment]
If it helps on `https://github.com/graasp/graasp/blob/main/src/services/auth/plugins/mobile/mobile.test.ts` at the end of the test file there is this test case : 
```typescript
it('Fail if token is invalid', async () => {
  const member = await saveMember();
  const token = jwt.sign({ sub: member.id }, 'INVALID_SECRET');
  const response = await app.inject({
    method: HttpMethod.Get,
    url: '/m/auth/web',
    query: { token },
  });
  expect(response.headers).not.toHaveProperty('set-cookie');
  expect(response.statusCode).toEqual(StatusCodes.UNAUTHORIZED);
});
```

**Comment by spaenleh** [comment]
fixed in #1166
